core/types/request.go

```diff
@@ -31,7 +31,8 @@ import (
 const WithdrawalRequestType byte = 0x01
 const DepositRequestType byte = 0x00
 const ConsolidationRequestType byte = 0x02
-const WithdrawalRequestDataLen = 76 // addr + pubkey + amt
+const ConsolidationRequestDataLen = 116 // addr + sourcePubkey + targetPubkey
```

Comment:
Duplicated in `consensus/misc/eip7251.go`

Reply:
Will remove either/both, going forward. Should do the cleanup after further simplification in next PRs